tests/language-feature/enums/unscoped-enum-explicit-type copy.slang

```diff
@@ -0,0 +1,27 @@
+//TEST(compute):COMPARE_COMPUTE(filecheck-buffer=CHECK): -shaderobj
+
+//TEST_INPUT:ubuffer(data=[0 0], stride=4):out,name=outputBuffer
+RWStructuredBuffer<uint32_t> outputBuffer;
+
+interface myType
```

Comment:
Can you leave a comment somewhere here to describe what problem this is testing?
Please provide a summary so that we don't need to read the whole code to understand the purpose of it.